packages/x-components/src/x-modules/facets/components/filters/all-filter.vue

```diff
@@ -16,16 +16,16 @@
 </template>
 
 <script lang="ts">
-  import Vue from 'vue';
-  import { Component, Prop } from 'vue-property-decorator';
+  import { computed, ComputedRef, defineComponent, PropType } from 'vue';
   import { Facet } from '@empathyco/x-types';
-  import { Getter, xComponentMixin } from '../../../../components';
+  import { Dictionary } from '@empathyco/x-utils';
   import BaseEventButton from '../../../../components/base-event-button.vue';
   import { isArrayEmpty } from '../../../../utils/array';
   import { VueCSSClasses } from '../../../../utils/types';
   import { XEventsTypes } from '../../../../wiring/events.types';
   import { FiltersByFacet } from '../../store';
   import { facetsXModule } from '../../x-module';
+  import { useGetter } from '../../../../composables';
```

Comment:
Try to avoid import composables from the barrel. We had issues with the vue3 integration in the past.